### PR TITLE
[MINOR] docs(lakehouse-iceberg): polish Iceberg REST service document

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
1. make it more clear about configuration and start Gravition server
2. add description to configurations.

### Why are the changes needed?
suggested by user

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
just documents

### CODE REVIEW

File: docs/iceberg-rest-service.md [hunk 1]
```diff
@@ -23,11 +23,19 @@ Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `
 Builds with Hadoop 2.10.x, there may compatibility issue when accessing Hadoop 3.x clusters.
 :::
 
-## How to start the Gravitino Iceberg REST catalog service
+## Gravitino Iceberg REST catalog service Configurations
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:26]: Heading should use sentence case and it should be "service configuration"

File: docs/iceberg-rest-service.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
 ### Capabilities
 
 - Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, supports all namespace and table interfaces. `Token`, `ReportMetrics`, and `Config` interfaces aren't supported yet.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:16]: and supports

File: docs/iceberg-rest-service.md [hunk 3]
```diff
@@ -23,11 +23,19 @@ Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `
 Builds with Hadoop 2.10.x, there may compatibility issue when accessing Hadoop 3.x clusters.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:23]: may be compatibility issues

File: docs/iceberg-rest-service.md [hunk 4]
```diff
@@ -23,11 +23,19 @@ Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `
...
-## How to start the Gravitino Iceberg REST catalog service
+## Gravitino Iceberg REST catalog service Configurations
 
-Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md).
+Assuming the Gravitino server is deployed in the `GRAVITINO_HOME` directory, you can locate the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md). There are three types of configurations for the Iceberg REST catalog service:
 
-### Gravitino Iceberg REST catalog service configuration
+1. [**REST Catalog Server Configuration**](#rest-catalog-server-configuration): you can specify the HTTP server properties like host and port.
+
+2. [**Iceberg Catalog Configuration**](#iceberg-catalog-configuration): you have the option to set the specified catalog-backend to either jdbc or hive.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:32]: Either use JDBC or Hive as they are proper names or abbreviations or put in code quotes if they are conjuration values.

File: docs/iceberg-rest-service.md [hunk 5]
```diff
@@ -55,7 +63,7 @@ You must set `gravitino.auxService.iceberg-rest.httpPort` explicitly, like `9001
 ### Iceberg catalog configuration
 
 :::info
-The Gravitino Iceberg REST catalog service using memory catalog for default. You can specify Hive or JDBC catalog for production environments.
+The Gravitino Iceberg REST catalog service using memory catalog for default. You can specify Hive or JDBC catalog in production environments.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:66]: The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using a Hive or JDBC catalog in a production environment.

OR 

The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using Hive or JDBC catalog for production environments.

**Review by justinmclean — COMMENTED** [review]
Needs some minor improvements.

File: docs/iceberg-rest-service.md (hunk 1 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 2 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 3 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 4 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: using `jdbc` and `hive`

File: docs/iceberg-rest-service.md (hunk 5 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md [hunk 6]
```diff
@@ -13,21 +13,29 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
 
 ### Capabilities
 
-- Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, supports all namespace and table interfaces. `Token`, `ReportMetrics`, and `Config` interfaces aren't supported yet.
-- Works as a catalog proxy, supporting `HiveCatalog` and `JDBCCatalog`.
+- Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, supports all namespace and table interfaces. `Token`, `ReportMetrics`, and `Config` interfaces aren't supports yet.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:16]: ", and supports all namespace"
aren't supported yet.

File: docs/iceberg-rest-service.md [hunk 7]
```diff
@@ -13,21 +13,29 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
...
+- Works as a catalog proxy, supporting `HiveCatalog` and `JDBCCatalog` for now.
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:17]: remove "for now".

File: docs/iceberg-rest-service.md [hunk 8]
```diff
@@ -13,21 +13,29 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
...
   doesn't support proxying to other HDFS users. See [How to access Apache Hadoop](gravitino-server-config.md) for more details.
 
 :::info
 Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `1` by default.
-Builds with Hadoop 2.10.x, there may compatibility issue when accessing Hadoop 3.x clusters.
+Builds with Hadoop 2.10.x, there may be compatibility issues when accessing Hadoop 3.x clusters.
 :::
 
-## How to start the Gravitino Iceberg REST catalog service
+## Gravitino Iceberg REST catalog service configurations
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:26]: configuration

File: docs/iceberg-rest-service.md [hunk 9]
```diff
@@ -13,21 +13,29 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
...
 
-Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md).
+Assuming the Gravitino server is deployed in the `GRAVITINO_HOME` directory, you can locate the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md). There are three types of configurations for the Iceberg REST catalog service:
```
Comment (justinmclean) [inline docs/iceberg-rest-service.md:28]: There are three configuration properties for the Iceberg REST catalog service:

**Review by justinmclean — COMMENTED** [review]
Just a couple of very minor things

File: docs/iceberg-rest-service.md (hunk 1 above)
Comment (justinmclean) [reply docs/iceberg-rest-service.md]: configuration not configurations

File: docs/iceberg-rest-service.md (hunk 2 above)
Comment (justinmclean) [reply docs/iceberg-rest-service.md]: see latest comment

File: docs/iceberg-rest-service.md (hunk 9 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 8 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 7 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 6 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

**Comment by jerryshao** [comment]
@FANNG1 do you need to merge this.

**Comment by FANNG1** [comment]
> @FANNG1 do you need to merge this.

updated

File: docs/iceberg-rest-service.md [hunk 10]
```diff
@@ -60,21 +71,21 @@ Gravitino provides a pluggable metrics store interface to store and delete Icebe
...
-### Iceberg catalog configuration
+### Gravitino Iceberg catalog backend configuration
 
 :::info
-The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify Hive or JDBC catalog for production environments.
+The Gravitino Iceberg REST catalog service uses the memory catalog by default. You can specify using a Hive or JDBC catalog in a production environment.
 :::
 
-#### Hive catalog configuration
+#### Hive catalog backend configuration
```
Comment (jerryshao) [inline docs/iceberg-rest-service.md:80]: Is it better changing to "Hive backend configuration"?

File: docs/iceberg-rest-service.md [hunk 11]
```diff
@@ -60,21 +71,21 @@ Gravitino provides a pluggable metrics store interface to store and delete Icebe
...
 
 | Configuration item                                  | Description                                                                                                 | Default value | Required   | Since Version |
 |-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------|---------------|------------|---------------|
 | `gravitino.auxService.iceberg-rest.catalog-backend` | The Catalog backend of Gravitino Iceberg REST catalog service, use the value **`hive`** for a Hive catalog. | `memory`      | Yes        | 0.2.0         |
 | `gravitino.auxService.iceberg-rest.uri`             | The Hive metadata address, such as `thrift://127.0.0.1:9083`.                                               | (none)        | Yes        | 0.2.0         |
 | `gravitino.auxService.iceberg-rest.warehouse `      | The warehouse directory of the Hive catalog, such as `/user/hive/warehouse-hive/`.                          | (none)        | Yes        | 0.2.0         |
 
-#### Iceberg JDBC backend configuration
+#### JDBC catalog backend configuration
```
Comment (jerryshao) [inline docs/iceberg-rest-service.md:88]: Also here.

File: docs/iceberg-rest-service.md (hunk 10 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md:80]: ok

Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

File: docs/iceberg-rest-service.md (hunk 11 above)
Comment (FANNG1) [reply docs/iceberg-rest-service.md]: done

**Review by jerryshao — APPROVED** [review]

**Pull request merged**